notebooks/data-sources/TestGrid/metrics/unexpected_test_failures.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>What does this output mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>If these outputs are meant to function as a "progress bar", I'd suggest just using tqdm instead</p>

Reply:
<p>My tqdm has never worked on the MOC jupyterhub ):</p>